prow/cluster/jobs/istio/api/istio.api.release-1.0.yaml

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
This is for `release-1.0` not `release-1.1`. Should this be included in this PR/commit?

Reply:
This was apparently missing for release-1.0. Updated PR desc.